OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -435,7 +440,7 @@ class AndroidICCAnalysis(val project: SomeProject) extends FPCFAnalysis {
                                         if (par.isObjectVariableInfo) {
                                             val cf = project.classFile(par.asObjectVariableInfo.clazz.asObjectType)
                                             if (cf.isDefined) {
-                                                val methods = cf.get.methods.filter(f ⇒ f.name != init && f.name != "<clinit>")
+                                                val methods = cf.get.methods.filter(f ⇒ !f.isConstructor && f.name != "<clinit>")
```

Comment:
Use f.isStaticInitializer instead of checking for "<clinit>" yourself. In fact, use f.isInitializer to check for both constructors and static initializers at once.